readme.rst

```diff
@@ -55,6 +55,17 @@ The command:
 the more standard ``make html``. For instance, ``PROJECT=azure make html`` will create the 
 azure related html pages in _build/azure.)
 
+Check compliance with contribution guides for each project 
+~~~~~~~~~~~
```

Comment:
Please ensure that these underlines are as long as the title they are underlining. Usually sphinx gives a warning if they are not, and our checks fail on warnings. 
Since this readme.rst is not part of any project, and our checks only build each project, the checks passed fine.